ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -118,49 +125,123 @@ ACP-103 introduced the following variables for calculating the gas price:
+When a block is produced which modifies $T$, both the consensus thread and the execution thread will update to the modified $T$ after their own handling of the block.
 
-#### Named blocks
+For example, restrictions of the queue size MUST be calculated based on the parent block's $T$.
 
-The Ethereum Mainnet APIs allow for retrieving blocks by named parameters that the API server resolves based on their consensus mechanism.
-Other than the _earliest_ (genesis) named block, which MUST be interpreted in the same manner, all other named blocks are mapped to SAE in terms of the execution status of all block transactions and MUST be interpreted as follows:
+Similarly, the time spent executing a block MUST be calculated based on the parent block's $T$.
 
- * _pending_: the most recently _accepted_ block;
- * _latest_: the block that was most recently _executed_;
- * _safe_ and _finalized_: the block that was most recently _settled_.
+### Block settlement
+
+For a _proposed_ block which includes timestamp $t_b$, all ancestors whose execution timestamp $t_e$ is $t_e + \tau \leq t_b$ are considered settled.
+
+The _proposed_ block must include the `stateRoot` produced by the execution of the most recently settled block.
+
+For any _newly_ settled blocks, the _proposed_ block must include all execution artifacts:
+- `receiptsRoot`
+- `logsBloom`
+- `gasUsed`
 
 > [!NOTE]
-> The finality guarantees of Snowman consensus remove any distinction between _safe_ and _finalized_. 
-> Furthermore, the _latest_ block is not at risk of re-org, only of a negligible risk of data corruption local to the API node.
+> If the block executor has fallen behind, the node may not be able to determine precisely which ancestors should be considered settled. If this occurs, validators MUST allow the block executor to catch up prior to deciding the block's validity.
+
+### Block validity and building
+
+After determining which blocks to settle, all remaining ancestors of the new block must be inspected to determine the worst-case bounds on $x$ and account balances. Account nonces are able to be known immediately.
+
+The worst-case bound on $x$ can be calculated by following the block executor update rules using $g_L$ rather than $g_C$.
+
+The worst-case bound on account balances can be calculated by charging the worst-case gas cost to the sender of a transaction along with deducting the value of the transaction from the sender's account balance.
+
+The `baseFeePerGas` field must be populated to the gas price based on the worst-case bound on $x$ at the start of block execution.
+
+### Configuration Parameters
+
+As noted above, SAE depends on the values of $\tau$ and $\lambda$ to be set as parameters and the value of $\omega$ is derived from $T$. 
+
+Parameters to specify for the C-Chain are:
+
+<div align="center">
+
+| Parameter | Description | C-Chain Configuration|
+| - | - | - |
+| $\tau$ | duration between execution and settlement | $5s$ |
```

Comment:
```suggestion
| $d$ | duration between execution and settlement | $5s$ |
| $\tau$ | TODO | $5s$ |
```

As per earlier comment decoupling the _definitions_ of $d$ and $\tau$.

Reply:
See above comment about $\omega$.